src/adagrams.js

```diff
@@ -1,15 +1,104 @@
+export const createLetterPool = () => {
+  const LETTERPOOL = {
+      "A":9,"B":2,"C":2,"D":4,
+      "E":12,"F":2,"G":3,"H":2,
+      "I":9,"J":1,"K":1,"L":4,"M":2,
+      "N":6,"O":8,"P":2,"Q":1,
+      "R":6,"S":4,"T":6,"U":4,
+      "V":2,"W":2,"X":1,
+      "Y":2,"Z":1
+  };
```

Comment:
A few things: 
- For true constants in JS, we use the [convention](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage) of all uppercase and underscores.

- These keys don't have to be strings we could have something like `A:9` (with no quotations). All your following logic will be the same.